tutorials/io/encrypting_save_games.rst

```diff
@@ -29,6 +29,13 @@ their own irrelevance would again take over their life.
 No, we definitely do not want this to happen, so let's see how to
 encrypt savegames and protect the world order.
 
+(NOTE: This method can not really prevent players from editing their savegames
```

Comment:
I'd move it at the end of the page to avoid spoiling the fun of the "Why?" section, and use the `.. note::` markup so that it appears as a note box.